src/test/java/org/broadinstitute/hellbender/utils/GenotypeUtilsUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.variant.variantcontext.*;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class GenotypeUtilsUnitTest {
+    private static final Allele Aref = Allele.create("A", true);
+    private static final Allele T = Allele.create("T");
+    private static final double DELTA_PRECISION = .001;
+
+
+    @DataProvider
+    public Object[][] getGTsWithAndWithoutLikelihoods(){
+        final GenotypeBuilder builder = new GenotypeBuilder("sample", Arrays.asList(Aref, T));
+        final List<Allele> DIPLOID_ALLELES = Arrays.asList(Aref, T);
+        final List<Allele> HAPLOID_ALLELES = Arrays.asList(Aref);
+        final int[] SOME_PLS = {0, 4, 1};
+        final int[] HAPLOID_PL = {0, 4};
+        return new Object[][]{
+                {builder.noPL().alleles(DIPLOID_ALLELES).make(), false},
+                {builder.noPL().alleles(HAPLOID_ALLELES).make(), false},
+                {builder.PL(HAPLOID_PL).alleles(HAPLOID_ALLELES).make(), false},
+                {builder.PL(SOME_PLS).alleles(DIPLOID_ALLELES).make(), true},
```

Comment:
Can you confirm that it's safe to re-use the same builder like this, even if the ordering of these test cases were to be scrambled?